### PR TITLE
Print Screen and Pause placement swap

### PR DESCRIPTION
**Pull request by trinity1561**
## Description

In the original json that was provided by the designer of the pcb had the the two button in the wrong row/column.  They were swapped.  This will update those 2 with the correct row and column.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Swapped the Print Screen and Pause button placement

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
Tagging @mechmerlin for review.

File: keyboards/keebio/quefrency/info.json [hunk 1]
```diff
@@ -169,6 +169,201 @@
               {"label":"Down", "x":15, "y":4},
               {"label":"Right", "x":16, "y":4}
           ]
+      },
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/info.json]: The changes to this file should be reverted.

Comment (trinity1561) [reply keyboards/keebio/quefrency/info.json]: I didn't mean to submit this as part of the PR.  I believe I reverted it in e0e20af.  Is there a way to delete this (62a226d)?

Comment (noroadsleft) [reply keyboards/keebio/quefrency/info.json]: The most straight-forward way, IMO:

```
git checkout a6857d18 -- keyboards/keebio/quefrency/info.json
git commit
git push
```

That'll edit the file so it matches the current QMK master version, and update this pull request.

**Comment by trinity1561** [comment]
I guess I messed up this PR with more commits than I wanted.  I'll try to fix those commits before redoing this PR.

**Pull request closed without merging**